pkg/reconciler/autoscaling/kpa/controller.go

```diff
@@ -19,6 +19,7 @@ package kpa
 import (
 	"context"
 
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
Shall we use https://github.com/dgraph-io/ristretto instead? It's tailored for fast concurrent performance which'd make this suitable for the activator (where it's dearly needed) as well.

Reply:
This is reconciler, we permit at most 2 threads at the same time. Not sure it warrants the complexity over simplicity